.github/workflows/ci.yml

```diff
@@ -52,8 +52,6 @@ jobs:
           - 'default'
           - 'delete'
         exclude:
-          - os: 'centos-6'
-            suite: 'delete'
           - os: 'ubuntu-1804'
```

Comment:
Might be worth adding a comment as to why these are excluded so we don't go through this again.